src/components/maps/GlobalMap.tsx

```diff
@@ -121,6 +144,7 @@ export const GlobalMap: React.FC<GlobalMapProps> = ({
         onDragStart={() => {
           setCursor('move')
         }}
+        onRender={onRender}
```

Comment:
I often see `onMove()` (instead of onRender) is used to track current view state.  Would it work here?
https://visgl.github.io/react-map-gl/docs/get-started/state-management#controlled-map

Reply:
1. Done
2. It is not possible to do that using URLSearchParams, as it automaticaly encodes special characters.
I probably can prevent that by manually construction the parameter. However, it takes off any benefits that URLSearchParams gives.

Therefore, I believe it will be more beneficial to use URLSearchParams and put up with percent encoded (%2f) symbols in the url.

But if you're ok with keeping '/' slash in the url, you can check out the changes that I made

Reply:
Oh, thank you. I was incorrectly reading documentation for [mapbox](https://github.com/mapbox/mapbox-gl-js) and  missed the fact that you're using maplibre. 

Fixed ✅